consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/messagewrappers/Proposal.java

```diff
@@ -74,7 +77,11 @@ public static Proposal decode(final Bytes data) {
     final RLPInput rlpIn = RLP.input(data);
     rlpIn.enterList();
     final SignedData<ProposalPayload> payload = SignedData.readSignedProposalPayloadFrom(rlpIn);
-    final Block proposedBlock = Block.readFrom(rlpIn, IbftBlockHeaderFunctions.forCommittedSeal());
+    final Block proposedBlock =
+        ProtocolRLPSpec.decodeBlockStandalone(
+            IbftProtocolSchedule.create(new StubGenesisConfigOptions()),
```

Comment:
Creating a stub genesis during a generic decode seems like (a) memory thrash for performance and (b) like we will lose config data if we depend on that for the object.  It feels like concerns are mixed improperly if we need to do this.